### DISCUSSION TITLE
New Error as of Yesterday: "Update Needed - To view this in your space update your google app" Link does not update

### DISCUSSION THREAD

**Discussion by MicDesign1** [google/model-viewer#2488] [Q&A]
When clicking 'view in your space' on GLB AR model, pop up states "Update Needed - To view this in your space, update your Google app" then links prompt "no thanks" or "update" .. When you click Update, it takes you to 'Google LLC' in the Play Store with Open or Uninstall. No 'update' available. What do I tell my users to update? Or how to better prompt it automatically?

**Comment by elalish** [comment]
It looks like this is a recent Scene Viewer bug that slipped out. Workaround for now is to use WebXR mode instead. Hopefully we'll get a Scene Viewer fix out shortly.

**Reply by MicDesign1** [reply]
Thank you!
